src/components/hooks/use-handle-signup.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react'
 import { navigate } from 'gatsby'
 import { useCountryRule } from './use-country-rule'
-import { isEuDomain } from 'common/utility'
+import { isEuDomain , isBrowser } from 'common/utility'
 import { DerivStore } from 'store'
 
 const useHandleSignup = (_is_ppc = false) => {
     const { is_loading, is_eu_location } = useCountryRule()
     const { setShowNonEuPopup } = React.useContext(DerivStore)
+    const currentLocation = isBrowser() && window.location.pathname.split('/').slice(0, 4).join('/')
```

Comment:
Why don't you use `getLanguage()` from utility insetead of this?

Reply:
@mahdiyeh-fs  because using this we have strange side effect. Language put in address line twice